crypto/conf/conf.c

```diff
@@ -642,6 +642,18 @@ int CONF_modules_load_file(const char *filename, const char *appname,
   return 1;
 }
 
+char *CONF_get1_default_config_file(void) {
+  const char *temp = "No support for Config files in AWS-LC.";
+  size_t temp_len = strlen(temp);
+
+  char *ret = (char *)OPENSSL_malloc(temp_len);
+  if(ret == NULL) {
+    OPENSSL_PUT_ERROR(CONF, ERR_R_MALLOC_FAILURE);
```

Comment:
suggest returning early here to guard against segfault on L653

Reply:
Ah my bad, forgot about that...

Reply:
You don't need to put the malloc failure message, [OPENSSL_malloc](https://github.com/aws/aws-lc/blob/main/crypto/mem.c#L201-L202) does that for you now.

Reply:
Ah sure, too much hygiene on my end